src/Kestrel.Core/Internal/Http2/Http2StreamFlowControl.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2
+{
+    public class Http2StreamFlowControl
```

Comment:
This looks like it's now response flow control specific and you'll need a different API for request flow control. Rename accordingly?